providers/aws/services/appstream/appstream_fleet_session_disconnect_timeout/appstream_fleet_session_disconnect_timeout.py

```diff
@@ -0,0 +1,30 @@
+from lib.check.models import Check, Check_Report
+from providers.aws.services.appstream.appstream_client import appstream_client
+
+max_disconnect_timeout_in_seconds = 5 * 60
+"""max_disconnect_timeout_in_seconds 300"""
```

Comment:
How about setting this variable in config.yaml and get it with get_config_var function?

Reply:
Done!